bin/src/_main.rs

```diff
@@ -139,7 +146,9 @@ pub async fn _main(
     let mut executor = Executor::new();
 
     let mut k8s_claimed_lease: Option<String> = None;
-    let (k8s_event_stream, metric_stats_source) = match create_k8s_client_default_from_env(
+    let mut metrics_stream_feature_meta: Option<FeatureLeaderMeta> = None;
+
+    let (k8s_event_stream, metric_stats_stream) = match create_k8s_client_default_from_env(

```

Comment:
I'm probably missing something but, is this also being defined below as well?

Reply:
lets chat about this more - but one is source, one is stream.

Reply:
Ah, yeah I see what you mean, my mistake. This looks good 👌